File: lib/monkey.rb
require "pry"
class Monkey
  @@foods_eaten = []
attr_accessor :eat_food, :name, :species
  def initialize name, species, eat_food
    @name = name
    @species = species
    @eat_food = eat_food
    @@foods_eaten.push(@eat_food)

  end

def eat
  @@foods_eaten
end
def introduce
  return " Monkey name #{@name}  species #{@species}  eaten food #{@eat_food}"
end
end



monkey1 = Monkey.new "tantalus", "Chlorocebus", "fruits"
monkey2 = Monkey.new "Mandrill", "Mandrillus", "ants"




binding.pry

puts "end of file"

File: lib/monkey.js
function Monkey(name, species) {
  this.name = name;
  this.species = species;
  this.foodsEaten = [];
};

Monkey.prototype.eat = function(food) {
  this.foodsEaten.push(food);
}

Monkey.prototype.introduce = function() {
  return "Hello! I'm " + this.name + " and I'm a " + this.species + " and I've eaten" + this.foodsEaten;
}


module.exports = Monkey;
